samples/snippets/src/main/java/pubsub/CreatePullSubscriptionExample.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pubsub;
+
+// [START pubsub_create_pull_subscription]
+
+import com.google.cloud.pubsub.v1.SubscriptionAdminClient;
+import com.google.pubsub.v1.ProjectSubscriptionName;
+import com.google.pubsub.v1.PushConfig;
+import com.google.pubsub.v1.Subscription;
+import com.google.pubsub.v1.TopicName;
+
+public class CreatePullSubscriptionExample {
+  public static void main(String... args) throws Exception {
```

Comment:
In the Node samples, we are actually pulling the three parameters out of args, but defaulting them to those TODO values. That way, it's possible to run the samples as part of a system-test (re: discussion at the meeting earlier).

Dunno if that's something we want to look at right now though.

Reply:
Never mind, I see down below the integration test.